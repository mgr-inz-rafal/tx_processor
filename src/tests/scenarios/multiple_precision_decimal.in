type,client,tx,amount
deposit,1,1,1
deposit,1,1,1.1
deposit,1,1,1.11
deposit,1,1,1.111
deposit,1,1,1.1111
deposit,1,1,1.11111
