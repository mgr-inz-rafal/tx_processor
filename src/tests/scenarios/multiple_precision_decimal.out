client,available,held,total,locked
1,6.54321,0,6.54321,false